vars/notify.groovy

```diff
@@ -14,13 +14,13 @@ import java.util.concurrent.TimeUnit
 //import vars.withSecrets
 
 
-// Used to set status to FAILURE and emit the failure reason to slack/email.
+// Used to emit the failure reason to slack/email.
 class FailedBuild extends Exception {
-   def statusToSet;
```

Comment:
You could get rid of this because we were always using the default of FAILURE in practice?